src/interaction-handlers/voteReminderButton.ts

```diff
@@ -0,0 +1,38 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import { Time } from '@sapphire/duration';
+import { InteractionHandler, InteractionHandlerTypes } from '@sapphire/framework';
+import type { ButtonInteraction } from 'discord.js';
+import { remindMeButtonDisabled } from '../lib/components/button';
+import { editInteractionResponse } from '../lib/discord/interaction';
+
+@ApplyOptions<InteractionHandler.Options>({ interactionHandlerType: InteractionHandlerTypes.Button })
+export class VoteReminderButton extends InteractionHandler {
+	public override parse(interaction: ButtonInteraction<'cached'>) {
```

Comment:
```suggestion
	public override parse(interaction: ButtonInteraction) {
```

Warning, this button will not be in a guild but in DM so we can't use the cache for this button